### PR TITLE
Update VOTING_MEMBERS.md with Maplibre-gl-js contributors

### PR DESCRIPTION
**Pull request by HarelM**
A few of the more dominant contributors in the last few month for maplibre-gl-js.

## Nominees

- [x] @chrneumann has approve this Pull Request
- [x] @chrneumann shared their email via form
- [x] @coliff has approve this Pull Request
- [x] @coliff has shared their email via form
- [x] @dennemark has approve this Pull Request
- [x] @dennemark has shared their email via form
- [x] @kubapelc has approve this Pull Request
- [x] @kubapelc has shared their email via form
- [x] @olsen232 has approve this Pull Request
- [x] @olsen232 has shared their email via form
- [x] @Pheonor has approve this Pull Request
- [x] @Pheonor has shared their email via form
- [x] @sbachinin has approve this Pull Request
- [x] @sbachinin has shared their email via form
- [x] @smellman has approve this Pull Request
- [x] @smellman has shared their email via form
- [x] @vicb has approve this Pull Request
- [x] @vicb has shared their email via form

### CODE REVIEW

**Comment by HarelM** [comment]
CC: @chrneumann @coliff, @dennemark, @kubapelc, @olsen232, @Pheonor, @sbachinin, @smellman, @vicb

Please let me know if I have missed someone.

**Comment by wipfli** [comment]
If you are one of the people who are nominated, and you would like to become a MapLibre Voting Member, then we need your email address for the voting system among other personal details. Please fill out this form: https://share-eu1.hsforms.com/1OcrNFreTRMqPRb0_PlOt3gfn2ab

CC: @chrneumann @coliff, @dennemark, @kubapelc, @olsen232, @Pheonor, @sbachinin, @smellman, @vicb

**Review by wipfli — APPROVED** [review]

**Review by coliff — APPROVED** [review]

**Review by louwers — COMMENTED** [review]
To help with the vote, here is a list of pull request for each nominee:

- **chrneumann**: https://github.com/maplibre/maplibre-gl-js/pulls/chrneumann
- **coliff**: https://github.com/maplibre/maplibre-gl-js/pulls/coliff
- **dennemark**: https://github.com/maplibre/maplibre-gl-js/pulls/dennemark
- **kubapelc**: https://github.com/maplibre/maplibre-gl-js/pulls/kubapelc
- **olsen232**: https://github.com/maplibre/maplibre-gl-js/pulls/olsen232
- **Pheonor**: https://github.com/maplibre/maplibre-gl-js/pulls/Pheonor
- **sbachinin**: https://github.com/maplibre/maplibre-gl-js/pulls/sbachinin
- **smellman**: https://github.com/maplibre/maplibre-gl-js/pulls/smellman
- **vicb**: https://github.com/maplibre/maplibre-gl-js/pulls/vicb

File: VOTING_MEMBERS.md [hunk 1]
```diff
@@ -30,10 +30,16 @@ The Voting Members, in alphabetic order by their GitHub handles, are:
...
 
+[@chrneumann](https://github.com/chrneumann)
+
+[@coliff](https://github.com/coliff)
+
 [@danielgard](https://github.com/danielgard) (komoot)
 
 [@dayjournal](https://github.com/dayjournal) (Mierune)
 
+[@dennemark](https://github.com/dennemark)
```
Comment (wipfli) [inline VOTING_MEMBERS.md:41]:
```suggestion
[@dennemark](https://github.com/dennemark) (Form Follows You)
```

**Review by smellman — APPROVED** [review]

**Review by Pheonor — APPROVED** [review]

**Review by sbachinin — APPROVED** [review]

**Review by vicb — APPROVED** [review]

**Review by kubapelc — APPROVED** [review]

**Review by dennemark — APPROVED** [review]

**Review by olsen232 — APPROVED** [review]

**Review by chrneumann — APPROVED** [review]

**Pull request merged**